packages/itwin/ec3-widget/src/components/EC3Router.tsx

```diff
@@ -22,7 +22,7 @@ export const EC3Router = ({ currentRoute, navigateTo, goBack }: EC3RouterProps)
           onClickCreate={() =>
             navigateTo(() => ({
               step: RouteStep.TemplateMenu,
-              title: "Create Template",
+              title: "Create Report",
```

Comment:
This might be confusing is user is also using the Reports config widget. User might expect this to navigate them to creating a Report using the Reporting API, not the EC3 Configuration template.

Reply:
done, renamed it back to Template